phpstan.neon

```diff
@@ -16,6 +16,9 @@ parameters:
       - translations/
     analyse:
       - *SocialProfileTrait.php
+  # Not sure we can specify generic types properly with Drupal coding standards
+  # yet, disable for now.
+  checkGenericClassInNonGenericObjectType: false
```

Comment:
https://github.com/mglaman/drupal-check/blob/main/phpstan.neon#L6
https://github.com/drupal-graphql/graphql/blob/8.x-4.x/phpstan.neon#L14

I've also been running in to the same issue, for now following the rest until we have a better idea of how to align this.